step-templates/ansible-run-playbook.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Id": "f0af011b-f99d-4809-84fe-af005b619fa2",
+  "Name": "Run Ansible Playbook",
+  "Description": "Step template to run an ansible playbook using bash",
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.Script.Syntax": "Bash",
+    "Octopus.Action.Script.ScriptBody": "ansible-playbook $(get_octopusvariable \"Ansible.Playbook.Path\")\nplaybookRC=$?\n\nif [ $playbookRC -ne 0 ]; then\n  exit $playbookRC;\nfi"
+  },
+  "Parameters": [
+    {
+      "Id": "c9c94c61-07a7-4e7e-a52c-fec79ecd50d6",
+      "Name": "Ansible.Playbook.Path",
```

Comment:
Nitpick: can you alter the variable name to `RunAnsible.Playbook.Path` - this is just on the slight chance we add more ansible steps, and someone else adds a parameter named similar :smile: 

Note: You'll also need to update the script too :)